CONTRIBUTING.md

```diff
@@ -40,14 +39,204 @@ If you happen to have a fix for a package, please have a look at the
 issue tracker to see if it doesn't concern an existing issue. If that
 is the case mention it in your pull request.
 
-
 Adding new packages
 -------------------
 
 Information about creating new packages and adding them to repository
 is available in
 [opam's manual](https://opam.ocaml.org/doc/Packaging.html).
 
+The following is a tutorial that provides a step-by-step procedure to
+make a `hello` package available in opam.ocaml.org. The top-level
+sources contain the following files with the opam file having the same
+name of the project.
+
+```
+$ ls hello/
+
+src/
+dune
+dune-project
+hello.opam
+```
+
+A quick summary of the steps to be followed are listed below:
+
+```
+$ dune build
+$ dune-release lint
+$ dune-release tag
+$ dune-release distrib
+$ dune-release publish
+$ dune-release opam pkg
+$ dune-release opam submit
```

Comment:
Is it worth adding (or replacing) that this can be replaced by just? Or is it worth detailing?
```
$ dune-release tag
$ dune-release
```